### PR TITLE
Drag and Drop Image Uploads

### PR DESCRIPTION
**Pull request by carsenk**
This adds in support for drag and drag in the textarea for posting on BitClout

It follows the current file upload rules and uses the same functionality as copy/paste or selecting the file regularly.

Preview:
e3256257bf

Follow me! https://bitclout.com/u/carsenk

https://bitclout.com/posts/8ab9370e55db4a1d00a63e6a9688f1e5237275e7a4f8d07a005948535df1736d

### CODE REVIEW

**Comment by carsenk** [comment]
@lazynina @maebeam @diamondhands0 ready for your review!

File: src/app/feed/feed-create-post/feed-create-post.component.ts [hunk 1]
```diff
@@ -86,6 +86,12 @@ export class FeedCreatePostComponent implements OnInit {
     }
   }
 
+  uploadFile(event: any): void {
+    console.log("Dropped Image");
```
Comment (lazynina) [inline src/app/feed/feed-create-post/feed-create-post.component.ts:90]: remove console logging

File: src/app/feed/feed-create-post/feed-create-post.component.ts [hunk 2]
```diff
@@ -86,6 +86,12 @@ export class FeedCreatePostComponent implements OnInit {
...
+
+    this._handleFileInput(event[0]);
```
Comment (lazynina) [inline src/app/feed/feed-create-post/feed-create-post.component.ts:92]: we should skip uploading if `isComment` is true to reduce image uploads that aren't going to be displayed

**Review by lazynina — CHANGES_REQUESTED** [review]

File: src/app/feed/feed-create-post/feed-create-post.component.ts (hunk 1 above)
Comment (carsenk) [reply src/app/feed/feed-create-post/feed-create-post.component.ts:90]: Ah yes

File: src/app/feed/feed-create-post/feed-create-post.component.ts (hunk 2 above)
Comment (carsenk) [reply src/app/feed/feed-create-post/feed-create-post.component.ts:92]: lololol....you know I want comments so bad hah, will do! lol

**Review by lazynina — APPROVED** [review]

**Review by maebeam — APPROVED** [review]
Looks good to me! Will let @lazynina merge

**Pull request merged**